lib/ios/AirMaps/AIRMapMarker.m

```diff
@@ -205,13 +205,52 @@ - (void)_handleTap:(UITapGestureRecognizer *)recognizer {
     
     if (marker.selected) {
         CGPoint touchPoint = [recognizer locationInView:marker.map.calloutView];
-        if ([marker.map.calloutView hitTest:touchPoint withEvent:nil]) {
+        CGRect bubbleFrame = [self.calloutView convertRect:marker.map.calloutView.bounds toView:marker.map];
+        CGPoint touchPointReal = [recognizer locationInView:self.calloutView];
+        
+        // moved to [AIRMap hitTest:withEvent:]
+//        BOOL isInsideCallout = [self.calloutView isPointInside:touchPointReal];
+//        if (!isInsideCallout) {
+//            CGPoint touchPointMap = [recognizer locationInView:marker.map];
+//            AIRMapMarker* markerAtTapPoint = [self.map markerAtPoint:touchPointMap];
+//            if (markerAtTapPoint != nil) {
+//                [markerAtTapPoint showCalloutView];
+//            } else {
+//                [self.map.calloutView dismissCalloutAnimated:YES];
+//            }
+//            return;
+//        }
+        
+        UIView *calloutMaybe = [marker.map.calloutView hitTest:touchPoint withEvent:nil];
```

Comment:
can we find a better name for `calloutMaybe`

Reply:
Sure, renamed to `calloutView`